torch/csrc/jit/serialization/export.cpp

```diff
@@ -864,6 +867,20 @@ unsigned long long int GraphEncoder::GetGraphProtoSize(
     size += tensor_proto->ByteSizeLong();
     size += tensor.element_size() * tensor.numel();
   }
+
+  // Add up all onnx::Constant nodes
+  for (const auto& node : graph->nodes()) {
+    if (node->kind() == ::c10::onnx::Constant) {
+      size_t tensor_size_bytes = 0;
+      // Get the tensor from the constant node
+      c10::optional<IValue> constant = node->t(attr::value);
```

Comment:
I recall `node->t()` returns at::Tensor.

Reply:
I suspect the CI failure is related. I'm not sure why, maybe undefined behavior due to typing issues.

Reply:
I couldn't find any `node->t()` or `node.t()` in the code-base

Reply:
fixed